pkg/yurttunnel/server/serveraddr/addr_test.go

```diff
@@ -0,0 +1,29 @@
+package serveraddr
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"fmt"
+)
+
+func TestGetDefaultDomainsForSvc(t *testing.T) {
+	domains := getDefaultDomainsForSvc("", "")
+	if len(domains) == 0 {
+		t.Log("domains len is zero")
+	}
+
+	ns :="hello"
+	name := "world"
+	domains = getDefaultDomainsForSvc(ns, name)
+	if len(domains) == 0 {
+		t.Log("domains len is zero")
+	} else {
+		if len(domains) == 4{
+			// expected, actual
+			assert.Equal(t, name, domains[0])
```

Comment:
maybe it does not good to import new lib(`github.com/stretchr/testify/assert`), how about only use `testing` to replace it?

Reply:
I use testing replace it